docs/GettingStarted/CreatingAProfile.md

```diff
@@ -0,0 +1,67 @@
+# Creating a Profile
+
+This page will walk you through creating basic profiles with which you can generate data.
+
+[Profiles](../Profiles.md) are JSON documents consisting of three sections, the schema version, the list 
+of fields and the rules.
+
+- **Schema Version** - Dictates the method of serialisation of the profile in order for the generator to 
+interpret the profile fields and rules. The latest version is v3.
+```
+    "schemaVersion": "v3",
+```
+- **List of Fields** - An array of column headings is defined with unique "name" keys.
+```
+    "fields": [
+        {
+            "name": "Column 1 Name"
```

Comment:
Can we rename `Column 1 Name` as `Column 1`, `Field 1` or better still something relevant in the real-world, like `Name` or `Age` etc